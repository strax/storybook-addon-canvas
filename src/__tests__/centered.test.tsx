import React from "react";
import background from "../background";
import { mount } from "enzyme";
import { centered } from "..";

const ctx = { kind: "test", story: "test" };

describe("centered", () => {
  test("snapshot", () => {
    const rendered = mount(centered(() => <div>Test case</div>, ctx) as JSX.Element);
    expect(rendered).toMatchSnapshot();
  });
});
