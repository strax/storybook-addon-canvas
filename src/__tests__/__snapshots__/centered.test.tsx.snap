// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`centered snapshot 1`] = `
<div>
  Test case
</div>
`;
